packages/components/src/molecules/ProductCard/ProductCardContent.tsx

```diff
@@ -136,6 +145,22 @@ const ProductCardContent = forwardRef<HTMLElement, ProductCardContentProps>(
           <DiscountBadge listPrice={listingPrice} spotPrice={sellingPrice} />
         )}
         {outOfStock && <Badge>{outOfStockLabel}</Badge>}
+        {deliveryBadges && (
+          <div data-fs-product-card-delivery-badges>
```

Comment:
suggestion: In addition to the tokens, I'd also change those data attributes to add `promise `... what do you think?
```suggestion
          <div data-fs-product-card-delivery-promise-badges>
```